read.go

```diff
@@ -108,7 +108,9 @@ func (q *MmapQueue) Dequeue() error {
 	offset = (offset + length) % q.conf.arenaSize
 	q.index.putHead(aid, offset)
 
-	return nil
+	q.mutateOpsSinceFlush++
```

Comment:
Not sure if we should increment this on read ops too.

Reply:
I considered updating the index (q.index.putHead(...)) as a mutateOp.